client/src/main/java/org/apache/nemo/client/DriverEndpoint.java

```diff
@@ -83,6 +83,6 @@ public DriverEndpoint(final PlanStateManager planStateManager,
    * @return the final state of this plan.
    */
   PlanState.State waitUntilFinish() {
-    return planStateManager.waitUntilFinish();
+    return waitUntilFinish(0, TimeUnit.MILLISECONDS);
```

Comment:
Same here. This function doesn't seem to be used any more.

Reply:
Ditto.